component/authenticator/src/test/java/org/wso2/carbon/identity/application/authenticator/totp/TOTPAuthenticatorTest.java

```diff
@@ -669,6 +680,10 @@ public void testInitiateAuthenticationRequestAdminEnforces()
     public void testInitiateAuthenticationWithEnableTOTP()
             throws AuthenticationFailedException, UserStoreException, IOException {
 
+//        Map<String, String> parameterMap = new HashMap<>();
+//        AuthenticatorConfig authenticatorConfig = new AuthenticatorConfig("totp", true,
+//                parameterMap);
+
```

Comment:
it the line is not needed, let's remove them

Reply:
Removed them in 4fbf32e44ce549a566a69d8d0947a7d9f7aae4a5 , as they are not needed